systemtest/src/test/java/io/strimzi/systemtest/security/oauth/OauthAuthorizationST.java

```diff
@@ -2,7 +2,7 @@
  * Copyright Strimzi authors.
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
-package io.strimzi.systemtest.oauth;
+package io.strimzi.systemtest.security.oauth;
```

Comment:
Security seems fine here, as a package name, as security is a pretty big topic and is mostly about ensuring you have no gaps and everything is correctly configured/encrypted etc